.vscode/launch.json

```diff
@@ -0,0 +1,586 @@
+                "--gtest_break_on_failure",
+                "--gtest_filter=CompositeValueSpecJsonTest.TestDefaultAndEmptySpacerRoundTrips"
+            ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+        },
+        {
+            "name": "Debug specific ECObjects test",
+            "type": "cppvsdbg",
+            "request": "launch",
+            "program": "${env:OutRoot}Winx64/Product/ECObjects-Gtest/ECObjectsTest.exe",
+            "args": [
+                "--gtest_break_on_failure",
+                "--gtest_filter=SchemaJsonSerializationTest.SchemaWithItems"
+            ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+        },
+        {
+            "name": "Debug specific ECPresentation test",
+            "type": "cppvsdbg",
+            "request": "launch",
+            "program": "${env:OutRoot}Winx64/Product/ECPresentation-Gtest/ECPresentationTest.exe",
+            "args": [
+                "--gtest_break_on_failure",
+                "--gtest_filter=CustomNodesProviderTests.ReturnsValidChildNode"
+            ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+        },
+        {
+            "name": "Debug specific Geomlib test",
+            "type": "cppvsdbg",
+            "request": "launch",
+            "program": "${env:OutRoot}Winx64/Product/GeomLibs-Gtest/GeomLibsTest.exe",
+            "args": [
+                "--gtest_break_on_failure",
+                "--gtest_filter=MSBsplineSurface.BadBoundaryMarkupOnSmallBsurf"
+            ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+        },
+        {
+            "type": "node",
+            "request": "attach",
+            "name": "Attach",
+            "port": 9229
+        },
+        {
+            "name": "Launch Native debugger for index.js tests",
+            "type": "cppvsdbg",
+            "request": "launch",
+            "program": "node.exe",
+            "args": [
+                "./iModelJsNodeAddon/api_package/ts/lib/test/index.js"
+            ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+        },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "imodel-native unit tests",
+            "program": "${workspaceFolder}/iModelJsNodeAddon/api_package/ts/lib/test/index.js"
+        },
+        {
+            "name": "(Windows) Attach",
+            "type": "cppvsdbg",
+            "request": "attach",
+            "processId": "${command:pickProcess}"
+        },
+        {
+            "name": "LNX Native BeHttpTest",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${env:OutRoot}LinuxX64/static/ProductCopy/BeHttp-Gtest/BeHttpTest",
```

Comment:
We should remove all of the dlls that no longer exist.

Reply:
Done.